content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,427 @@
+
+.. note::
+   Installing *just* the *Events* application on a trial database installs additional applications
+   automatically: :guilabel:`Discuss`, :guilabel:`Sales`, :guilabel:`Dashboards`,
+   :guilabel:`Invoicing`, and :guilabel:`Website`.
+
+Add or modify the cost of the ticket in the :guilabel:`Price` field.
+
+.. note::
+   It should be noted that a modified ticket price on the event form, in the :guilabel:`Tickets`
+   tab, does **not** change the price that appears on the ticket's product form. This modification
+   is specific to the event being configured.
+
+Next, determine a :guilabel:`Sales Start` and :guilabel:`Sales End` date in their respective fields.
+To do that, click into the blank field to reveal a calendar popover. From there, select the desired
+date and time, then click :icon:`fa-check` :guilabel:`Apply`.
+
+Then, if desired, designate a :guilabel:`Maximum` amount of that specific ticket that can be sold.
+
+The :guilabel:`Taken` column will populate with accurate data as tickets are sold.
+
+In the :guilabel:`Color` column, users have the option to add a custom color code to different
+tickets.
+
+To delete any tickets from the :guilabel:`Tickets` tab, click the :icon:`fa-trash-o`
+:guilabel:`(trash can)` icon on the corresponding line for the ticket that should be deleted.
+
+.. tip::
+   To add an optional :guilabel:`Description` column to the :guilabel:`Tickets` tab, click the
+   :guilabel:`additional options` :icon:`oi-settings-adjust` menu, located to the far-right of the
+   column titles.
+
+   Then, tick the checkbox beside :guilabel:`Description` from the resulting drop-down menu.
+
+   When added, the option to add brief descriptions for each event ticket appears, which can be used
+   to inform attendees of any perks or amenities that may coincide with specific ticket purchases.
+
+.. _create_events/event_form/communication:
+
+Communication tab
+-----------------
+
+In the :guilabel:`Communication` tab of an event form, create various marketing communications that
+can be scheduled to be sent at specific intervals leading up to, and following, the event.
+
+.. image:: create_events/communication-tab.png
+   :align: center
+   :alt: Typical communication tab on an event form in the Odoo Events application.
+
+.. note::
+   By default, Odoo provides three separate pre-configured communications on every new event form.
+   One is an email sent after each registration to confirm the purchase with the attendee. The other
+   two are email event reminders that are scheduled to be sent at different time intervals leading
+   up to the event to remind the recipient of the upcoming event.
+
+To add a communication in the :guilabel:`Communication` tab, click :guilabel:`Add a line`. Then,
+select the desired type of communication in the :guilabel:`Send` field. The options are:
+:guilabel:`Mail`, :guilabel:`SMS`, :guilabel:`Social Post`, or :guilabel:`WhatsApp`.
+
```

Comment:
To also cut down on the number of admonitions mixed with instructions in this section, I am suggesting the following
```suggestion
There is no limit to the number of communications that can be added in the :guilabel:`Communication`
tab of an event form.

To delete a communication from the :guilabel:`Communication` tab, click the :icon:`fa-trash-o`
:guilabel:`(trash can)` icon on the corresponding communication line. Doing so removes the
communication from the event entirely.

```